content/developer/writing_plugins/creating-your-own-plugin.md

```diff
@@ -182,7 +171,7 @@ def stop(**kwargs):
         ctx.logger.info('HTTP server is not running!')
 {{< /highlight >}}
 
-Runtime properties are saved in Cloudify storage after the plugin's operation invocation is complete. (For which the `@operation` decorator is responsible).
+Runtime properties are saved in Cloudify storage after the plugin's operation invocation is complete.
 
 Where it is important to immediately save runtime properties to Cloudify storage, call the `ctx.update` method.
 
```

Comment:
I failed to mention that I heard someone say we might start requiring people to use the Operation decorator again. Ask @idob-cloudify

Reply:
I did. It's not going to be mandatory, but it will be required for certain features (such as resumable operations). Seems like it'd be a good idea to have people get in the habit of using `@operation` anyway. I will update the PR's.